lib/common/tests/options/pcmk__env_option_enabled_test.c

```diff
@@ -27,11 +27,11 @@ enabled_true_value(void **state)
     pcmk__mock_getenv = true;
 
     expect_string(__wrap_getenv, name, "PCMK_env_var");
-    will_return(__wrap_getenv, "true");
+    will_return(__wrap_getenv, PCMK_VALUE_TRUE);
     assert_true(pcmk__env_option_enabled(NULL, "env_var"));
 
     expect_string(__wrap_getenv, name, "PCMK_env_var");
-    will_return(__wrap_getenv, "true");
+    will_return(__wrap_getenv, PCMK_VALUE_TRUE);
```

Comment:
This test should not be changed, since it is a test that the literal string "true" is treated as true (effectively, it is also testing that PCMK_VALUE_TRUE is defined correctly)